pyccel/codegen/printing/cwrappercode.py

```diff
@@ -273,88 +268,187 @@ def get_cast_function_call(self, cast_type, arg):
 
         return FunctionCall(cast_function, [arg])
 
+    # --------------------------------------------------------------------
+    # Functions that take care of collecting (data type, rank) checks and creating the error code
+    # --------------------------------------------------------------------
+    def _generate_TypeError_message(self, variable):
+        """
+        Generate TypeError message from the variable informations (datatype, precision)
```

Comment:
Grammar : plural of `information` is `information`

Reply:
done 0da2e5b